app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -1,28 +1,30 @@
+<p>A teaching internship gives you the opportunity to spend 3 weeks in a school and find out what teaching is really like. You’ll experience being in a classroom, gain new skills, and be paid £300 per week.</p>
 
-<p>You may be eligible for a paid teaching internship if you're:</p>
+<p>Teaching internships take place during June and July this year. Places fill up quickly.</p>
+
+<h2>Subjects you can do your internship in</h2>
+
+<p>Teaching internships are only available for certain subjects. You can do an internship in:</p>
 
 <ul>
- <li>in any year of a science, technology, engineering or maths (STEM) undergraduate degree</li>
-  <li>interested in teaching secondary maths or physics</li>
+ <li>computing</li>
+  <li>maths</li>
+  <li>physics</li>
+  <li>languages (north-east schools only)</li>
 </ul>
 
-<p>You'll experience the teaching profession, get new skills and be paid £300 per week.</p>
-
-<p><strong>The Teaching Internship Programme has now closed for 2021. Please check back on this page for future updates.</strong></p>
+<p>To take part, you’ll need to be studying for a degree related to the subject you want to teach (contact your chosen school if you’re unsure whether you meet this criteria).</p>
```

Comment:
Do we need brackets in this sentence or could 'Contact your chosen school' just be a new sentence without brackets?'